configs/components/facter.json

```diff
@@ -1 +1 @@
-{"url": "git://github.com/puppetlabs/facter.git", "ref": "8150368f38349f1dbf44e25a238c5e3d00dcda21"}
+{"url": "git://github.com/puppetlabs/facter.git", "ref": "7c86863931e83ce9215aa35ccca7780ba33029fc"}
```

Comment:
I confess I am slightly out of the loop, but should we be using a SHA here or should we be using a tag or branch name?